packages/syft/src/syft/service/code/status_service.py

```diff
@@ -91,5 +92,15 @@ def get_all(
             return result.ok()
         return SyftError(message=result.err())
 
+    @service_method(path="code_status.remove", name="remove", roles=ADMIN_ROLE_LEVEL)
+    def remove(
+        self, context: AuthedServiceContext, uid: UID
+    ) -> SyftSuccess | SyftError:
+        """Remove a user code item status"""
+        result = self.stash.delete_by_uid(context.credentials, uid=uid)
```

Comment:
maybe we also need to remove from the code version from the code history service?